app_dart/test/request_handlers/udpate_agent_health_history_test.dart

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:cocoon_service/cocoon_service.dart';
+import 'package:cocoon_service/src/model/appengine/agent.dart';
+import 'package:cocoon_service/src/request_handlers/update_agent_health_history.dart';
+import 'package:cocoon_service/src/request_handling/body.dart';
+import 'package:cocoon_service/src/service/datastore.dart';
+import 'package:googleapis/bigquery/v2.dart';
+import 'package:test/test.dart';
+
+import '../src/bigquery/fake_tabledata_resource.dart';
+import '../src/datastore/fake_cocoon_config.dart';
+import '../src/datastore/fake_datastore.dart';
+import '../src/request_handling/fake_authentication.dart';
+
+void main() {
```

Comment:
There should be another test for when an error occurs uploading to bigtable (check to see if the error was logged). Also, a test for agents that are not visible.

Reply:
Added test for log.info instead.
Existing test has considered the hidden case. Please refer to agent `windows1`.

Reply:
That is correct. I misread `reportedAgents` for `expectedAgents`.